content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -160,12 +172,21 @@ For detailed PowerFlex installation procedure, see the _Dell PowerFlex Deploymen
    | replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, excess pods will become stay in a pending state. Defaults are 2 which allows for Controller high availability. | Yes | 2 |
    | storageCapacity.enabled | Enable/Disable storage capacity tracking | No | true |
    | storageCapacity.pollInterval | Configure how often the driver checks for changed capacity | No | 5m |
+   | nfsAcls | enables setting permissions on NFS mount directory | No | 0777 |
+   | externalAccess | allows to specify additional entries for host to access NFS volumes. Both single IP address and subnet are valid entries | Yes | none |
```

Comment:
remove the instances for external access

Reply:
done